src/components/AttendeeModal/AttendeeModal.tsx

```diff
@@ -0,0 +1,122 @@
+            }));
+            closeModal();
+        }
+    };
+
+    return createPortal(
+        <>
+            <div className="modal-background" onClick={closeModal}></div>
+
+            <div className="attendee-modal">
+                <button type="button" id="close-dialog-btn" onClick={closeModal}>
+                    <div className="slant-lines">
+                        <div className="ltr"></div>
+                        <div className="rtl"></div>
+                    </div>
+                </button>
+
+                <div className="add-attendee">
+                    <h2>Add Attendee</h2>
+                    <ul className="attendee-form">
+                        <li className="input-wrapper">
+                            <label>
+                                Name <span>*</span>
+                            </label>
+                            <input type="text" name="name" onChange={handleOnChange} placeholder="Enter name" />
+                        </li>
+                        <li className="input-wrapper">
+                            <label>
+                                Email <span>*</span>
+                            </label>
+                            <input type="text" name="email" onChange={handleOnChange} placeholder="Enter email" />
+                        </li>
+                        <li className="form-buttons">
+                            <button type="button" className="reset-btn" onClick={() => setAttendee({ name: "", email: "" })}>
+                                Reset
+                            </button>
+                            <button type="button" className="submit-btn" onClick={handleOnClickAdd}>
+                                Add
+                            </button>
+                        </li>
+                    </ul>
+                </div>
+
+                {attendees.length !== 0 && (
+                    <>
+                        <div className="separator">
+                            <div className="line"></div>
+                            <h4>OR</h4>
+                            <div className="line"></div>
+                        </div>
+
+                        <div className="select-attendee">
+                            <h2>Select an Attendee</h2>
+                            <div className="input-wrapper">
+                                <select name="attendee" defaultValue="" onChange={handleOnSelection}>
+                                    <option value="" disabled>
+                                        --Select--
+                                    </option>
+                                    {attendees.map(({ name, email }: Attendee, idx: number) => (
+                                        <option key={`attendee-modal-${email}-${idx}`}>{name}</option>
+                                    ))}
+                                </select>
+                            </div>
+                        </div>
```

Comment:
create a separate function and render here, avoid using conditions directly inside return statement